1 tables.sql

```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
+go
+use TrackGermDB
+go
+create table dbo.germ(

```

Comment:
missing a primary key

Reply:
see previous, my latest commit addresses this also